docs/quickstart.md

```diff
@@ -92,6 +92,15 @@ For advanced setup please refer to [cookbook](./cookbook.md)
          }
          ```
 
+      3. If you are using ProGuard:
+         in `proguard-rules.pro` add
+
+         ```java
+         -keepclassmembers class [YOUR_GOOGLE_APPLICATION_NAME].BuildConfig {
+            public static <fields>;
+         }
```

Comment:
is this the literal fragment that needs to be copypasted or does it need to list every single field?

Reply:
can `https://developer.android.com/studio/build/shrink-code#keep-code` this be used instead?

```
-keep public class MyClass
```